src/python/pants_release/generate_github_workflows.py

```diff
@@ -1036,22 +1052,42 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Make GitHub Release",
+                    "run": dedent(
+                        """\
+                        GH_RELEASE_ARGS=()
+                        if [[ "${{ steps.get_info.outputs.is-release }}" == "true" ]]; then
+                            GH_RELEASE_ARGS+=(--title "${{ steps.get_info.outputs.build-ref }}")
+                            RELEASE_TAG="${{ steps.get_info.outputs.build-ref }}"
+                            if [[ ${RELEASE_TAG#release_} =~ [[:alpha:]] ]]; then
+                                GH_RELEASE_ARGS+=(--prerelease)
+                            fi
+                        else
+                            GH_RELEASE_ARGS+=(--title "dev_${{ steps.get_info.outputs.build-ref }}")
+                            GH_RELEASE_ARGS+=(--prerelease)
+                        fi
+
+                        gh release create "${{ steps.get_info.outputs.build-ref }}" "${GH_RELEASE_ARGS[@]}"
```

Comment:
I'm not sure the behavior if a release already exists. That may be the one missing piece here. If a release fails and we re-kick based on the tag this will try and create a release where one exists.

Will it error or just noop?

Reply:
OK, first, you need both `--title` and `--notes` for it to be non-interactive.

Second:
```console
josh@cephandrius:~/work/ghatest$ gh release create joshtag  --title rel1 --notes ""
https://github.com/thejcannon/ghatest/releases/tag/joshtag
josh@cephandrius:~/work/ghatest$ gh release create joshtag  --title rel1 --notes ""
HTTP 422: Validation Failed (https://api.github.com/repos/thejcannon/ghatest/releases)
Release.tag_name already exists
$ echo $?
1
```